README.md

```diff
@@ -72,7 +72,7 @@ Run inference:
 python generate.py --prompt "Hello, my name is"
 ```
 
-This will run the 7B model and require ~26 GB of GPU memory (A100 GPU).
+This will run the 3B pre-trained model and require ~7 GB of GPU memory (A100 GPU with bf16).
```

Comment:
This is already compatible with consumer devices, so I would remove the reference to A100 here, and remove the reference to `bfloat16` in the next section.

I'd also change the title of the next section to "small consumer devices" or something

Reply:
Done. I removed the quantization section entirely because it's not working yet